### ISSUE TITLE
Access to youtube data labelled by the IDM

### ISSUE THREAD

**Issue by roger-creus** [openai/Video-Pre-Training#42]
Hi!

I am wondering if you released the YouTube dataset with the labels given by the IDM. If so, how could we access it?

Thx!

**Comment by Miffyli** [comment]
Hey. Unfortunately the original pretraining set of VPT was not released in any form. The only data that was released as part of VPT was the contractor data, and later the BASALT competition data, both of which are listed in the README.md.

**Comment by brandonhoughton** [comment]
You may already know this but MineDojo released a YouTube index if you just want a large collection of Minecraft gameplay!

**Comment by roger-creus** [comment]
Hey! Thanks for the info. The MineDojo dataset is nice but it's not action-labelled! It should be possible to use VPT's IDM to label it right?

**Comment by Miffyli** [comment]
@roger-creus Potentially yes: VPT's IDM has generally quite robust to different preprocessing, but might not be perfect. You could also try different techniques to make use of the data (e.g., like what MineDojo did with MineCLIP, or what follow-up papers have done).
